### PR TITLE
[STORM-1019][STORM-1020] Added missing version info to make-maven-plugin & add javadoc to Fields & ITuple

### PR DESCRIPTION
**Pull request by rmkellogg**

### CODE REVIEW

File: storm-core/pom.xml [hunk 1]
```diff
@@ -849,6 +849,7 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>make-maven-plugin</artifactId>
+						<version>1.0-beta-1</version>
```
Comment (harshach) [inline storm-core/pom.xml]: not indented

**Comment by harshach** [comment]
+1. Can you fix the title of the JIRA as the pr contains doc fixes as well and minor whitespace issue.

**Comment by rmkellogg** [comment]
Sorry I had meant this as two distinct pull requests: STORM-1019 and STORM-1020.

Whitespace change was made simply to force Travis-CI to attempt rebuild.  It fails for odd reasons yesterday.

**Comment by rmkellogg** [comment]
Can we merge this?  No idea why Travis-CI is failing.  It worked earlier.

**Comment by rmkellogg** [comment]
Pull into distinct PRs.

**Pull request closed without merging**